app/models/financial_transaction_type.rb

```diff
@@ -27,4 +29,10 @@ def self.has_multiple_types
   def restrict_deleting_last_financial_transaction_type
     raise I18n.t('model.financial_transaction_type.no_delete_last') if FinancialTransactionType.count == 1
   end
+
+  private
+
+  def update_balance_of_ordergroups
+    ordergroups.each { |og| og.update_balance! }
+  end
```

Comment:
I suppose this only really needs to happen when the `financial_transaction_class` changes - or perhaps I'm missing something?

Reply:
yes